pages/desktop/base.py

```diff
@@ -248,6 +248,7 @@ def click_edit_profile(self):
                 click().perform()
 
             from pages.desktop.user import EditProfile
+            WebDriverWait(self.selenium, self.timeout).until(lambda s: self.selenium.title)
```

Comment:
If the intent here is to wait for the page to be fully loaded, this isn't necessarily going to do it. This is only going to wait until selenium sees a valid title for the page, which might still be too soon. I suggest removing this code, and adding the wait into the methods that interact with the checkbox.

In issue #764 @davehunt suggests waiting for the checkbox to be displayed and to have the correct state, so that code really belongs where we are doing the click. See my comments below.